### PR TITLE
add realtime consumption exception to LL consumer data manager metrics

### PR DESCRIPTION
**Pull request by walterddr**

### CODE REVIEW

File: pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java [hunk 1]
```diff
@@ -489,6 +489,9 @@ protected boolean consumeLoop()
     }
 
     if (_numRowsErrored > 0) {
+      _serverMetrics.addMeteredGlobalValue(ServerMeter.REALTIME_CONSUMPTION_EXCEPTIONS, _numRowsErrored);
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java:492]: I don't think this is the correct place to update the meter. This meter is used to track the consume exception, `_numRowsErrored` is used to track the invalid rows. IMO we should update the meter under the catch on line 422 and 425

Comment (walterddr) [reply pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java:492]: good idea. will do so

File: pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java [hunk 2]
```diff
@@ -420,6 +423,9 @@ protected boolean consumeLoop()
         _endOfPartitionGroup = messageBatch.isEndOfPartitionGroup();
         _consecutiveErrorCount = 0;
       } catch (PermanentConsumerException e) {
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java:425]: (minor) Suggest emitting in one place for readability
```suggestion
      } catch (Exception e) {
        _ServerMetrics.addMeteredGlobalValue...
        if (e instanceof PermanentConsumerException) {
          ...
        } else {
          ...
        }
      }
```

**Review by Jackie-Jiang — APPROVED** [review]
LGTM otherwise

File: pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java (hunk 2 above)
Comment (walterddr) [reply pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java:425]: will follow up in separate PR if we have more metrics

**Pull request merged**